databricks-sdk-java/src/test/java/com/databricks/sdk/DatabricksAuthLoadTest.java

```diff
@@ -0,0 +1,73 @@
+// These are auto-generated tests for Unified Authentication
+// In case of editing this file, make sure the change is propagated to all Databricks SDK codebases
+
+package com.databricks.sdk;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.databricks.sdk.core.ConfigResolving;
+import com.databricks.sdk.core.DatabricksConfig;
+import com.databricks.sdk.core.commons.CommonsHttpClient;
+import com.databricks.sdk.core.utils.GitHubUtils;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.*;
+import org.junit.jupiter.api.Test;
+
+public class DatabricksAuthLoadTest implements GitHubUtils, ConfigResolving {
```

Comment:
Let's leave a comment here about this test.

Also small suggestion: let's move this into a `benchmark` package. It'll be a new convention within Java SDK tests to keep things a bit more organized for now.

Reply:
https://github.com/databricks/databricks-sdk-java/pull/355/commits/68ae507c8f0888d60d5bcac44c809413aa5dec18